engine/cdn/storage/storageunit.go

```diff
@@ -205,8 +247,7 @@ func Init(ctx context.Context, m *gorpmapper.Mapper, db *gorp.DbMap, config Conf
 			}
 		}
 
-		schedulerEntry, err := scheduler.AddFunc(cfg.CronExpr, runFunc)
-		if err != nil {
+		if _, err := scheduler.AddFunc(cfg.CronExpr, runFunc); err != nil {
```

Comment:
Just above this, missing Swift storage initialization

Reply:
done